x-pack/platform/plugins/shared/streams/server/lib/streams/component_templates/logs_layer.ts

```diff
@@ -19,3 +29,186 @@ export const logsSettings: IndicesIndexSettings = {
     },
   },
 };
+
+export const baseFields: FieldDefinition = {
```

Comment:
Do you know why we don't store [`trace_flags`](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-traceflags)? It is optional but I think we drop it in the exporter.